File: app/src/main/java/com/wxm/camerajob/data/define/TakePhotoParam.java
package com.wxm.camerajob.data.define;

import java.sql.Timestamp;
import java.util.Calendar;
import java.util.Locale;

/**
 * 拍照参数
 * Created by 123 on 2016/6/13.
 */
public class TakePhotoParam {
    private final static int  WAIT_MSECS = 8000;

    public String   mPhotoFileDir;
    public String   mFileName;

    public String   mTag;
    public long     mWaitMSecs;

    public Timestamp    mTS;

    public TakePhotoParam(String pp, String fn, String tag) {
        mWaitMSecs = WAIT_MSECS;

        mTag = tag;
        mFileName = fn;
        mPhotoFileDir = pp;

        mTS = new Timestamp(Calendar.getInstance(Locale.CANADA).getTimeInMillis());
    }


    public TakePhotoParam() {
        mTS = new Timestamp(Calendar.getInstance(Locale.CANADA).getTimeInMillis());
    }
}

File: app/src/main/java/com/wxm/camerajob/data/define/TakePhotoParam.kt
package com.wxm.camerajob.data.define

import java.sql.Timestamp
import java.util.Calendar
import java.util.Locale

/**
 * param for take photo
 * Created by 123 on 2016/6/13.
 */
class TakePhotoParam(pp: String, fn: String, tag: String) {
    var mPhotoFileDir: String = pp
    var mFileName: String = fn
    var mTag: String = tag

    var mTS: Timestamp = Timestamp(System.currentTimeMillis())
}
